geom/util.go

```diff
@@ -69,15 +70,15 @@ func sequenceToXYs(seq Sequence) []XY {
 
 // fastMin is a faster but not functionally identical version of math.Min.
 func fastMin(a, b float64) float64 {
-	if a < b {
+	if math.IsNaN(a) || a < b {
```

Comment:
Nice, looks to also be consistent behaviour with the built-in `min` (and `max`) funcs offered in Go 1.21.

Do we need to handle `+-Inf` as well?

Reply:
Yes, once I upgrade to go 1.21, we _might_ not need these anymore. Probably not going to upgrade though until Go 1.23 comes out, in order to not force users to upgrade early.

It doesn't need to handle infinity _explicitly_, since it's handled implicitly by the `a < b` check. All of the cases involving infinity "just work" 😁.